src/utils.js

```diff
@@ -78,6 +78,13 @@ const upload = async (opts) => {
   const response = await fetch(endpoint, { method: 'POST', headers, body });
   const uri = await response.text();
 
+  if (!uri) {
+    console.log(response);
+    throw new Error(
+      'Malformed response; please report at https://github.com/iterative/cml/issues/443'
```

Comment:
Not very great for me to put links in the comment. Why not just report the request incidence?

Reply:
👌